bazel/repository_locations.bzl

```diff
@@ -64,11 +64,11 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "envoy-build-tools",
         project_desc = "Common build tools shared by the Envoy/UDPA ecosystem",
         project_url = "https://github.com/envoyproxy/envoy-build-tools",
-        version = "1162be3669036d2c09359a95d39ff65fc6608f39",
-        sha256 = "8f5ac011a443649a27a7b82bc447de4f564f9cb5b6812d87c3bc1b1e74d2055f",
+        version = "92823669a6005a14958795356e7cf59b90472ee3",
+        sha256 = "f36baf4ff1815a0682119d09965b8aadfe67b550a9121a3d110d68e15a61fd08",
```

Comment:
Can we use latest consider this isn't passing CI yet?
`version = "074db8931da210585fad66e62ee6a63095c295d4",`
`sha256 = "ab44d9c7505dcb60a24360d0cb4a6478e3766e9ed40ed3ce183557bd2dcadbcf",`
`release_date = "2022-02-16",`